qtawesome/__init__.py

```diff
@@ -81,6 +82,11 @@ def _instance():
     return _resource['iconic']
 
 
+def resetCache():
```

Comment:
```suggestion
def reset_cache():
```

This follows our convention of using camel case names in general for classes and not functions.